src/requirements/data/templates/ece.ts

```diff
@@ -0,0 +1,86 @@
+import { Template } from '@/requirements/types';
+
+const eceTemplate: Template = [
+  {
+    reqGroup: 'UNI',
+    name: 'Physical Education',
+    placeholderSemesters: [1, 2],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Mathematics',
+    placeholderSemesters: [1, 2, 3, 4],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Physics',
+    placeholderSemesters: [2, 3, 4],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Chemistry',
+    placeholderSemesters: [1],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'First-Year Writing Seminars',
+    placeholderSemesters: [1, 2],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Advanced Programming/Computer Engineering',
```

Comment:
@einc I believe this requirement is a check-off requirement as it does not show up in the engineering handbook flowchart and says this on the linked website in our ece file. If that is correct, @Aryan77 could you set the requirement in the ECE file to be double countable, and comment out this object in this templates file?

![image](https://user-images.githubusercontent.com/25535093/142490681-d0f1f598-7c63-4594-82df-4a57968b7601.png)

Reply:
Yes, there were a number of inconsistencies between what the handbook says and what the requirements in the ECE file says. I wrote the template to match exactly with the data we currently had - I will update both the ECE major file and the template files accordingly. Thanks!